src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Implementation/Providers/GetAzureProviderCmdlet.cs

```diff
@@ -89,5 +91,38 @@ public override void ExecuteCmdlet()
                 this.WriteObject(providers, enumerateCollection: true);
             }
         }
+
+        private PSResourceProvider[] ListPSResourceProviders()
```

Comment:
Sounds good

---
In reply to: [65991244](https://github.com/Azure/azure-powershell/pull/2380#discussion_r65991244) [](ancestors = 65991244)